### PR TITLE
[NETPATH-297] Update timeout to be per hop instead of per path

### PR DESCRIPTION
**Pull request by ken-schneider**
### What does this PR do?
This changes the behavior of `timeout` in the configuration of Network path. Previously, the `timeout` field controlled the timeout for the entire path trace. The default timeout has been updated to be `1000` which translates to 1 second per hop.

### Motivation
Other traceroute tools use this to configure their timeouts, some call this value "waittime". In the future this should allow us to more easily change the underlying implementation e.g. running traceroute serially or in parallel without the timeout having different meanings depending on the implementation.

### CODE REVIEW

**Review by jmw51798 — APPROVED** [review]

**Review by AyyLam — APPROVED** [review]

**Review by zoedt — APPROVED** [review]

**Review by jeremy-hanna — APPROVED** [review]
👍 for ASC owned files

File: cmd/agent/dist/conf.d/network_path.d/conf.yaml.example [hunk 1]
```diff
@@ -5,11 +5,11 @@ init_config:
     #
     # min_collection_interval: 60
 
-    ## @param timeout - integer - optional - default: 10000
-    ## Specifies how much time the full traceroute should take
-    ## in milliseconds
+    ## @param timeout - integer - optional - default: 1000
+    ## Specifies how much time in milliseconds traceroute should
```
Comment (alai97) [inline cmd/agent/dist/conf.d/network_path.d/conf.yaml.example:9]:
```suggestion
    ## Specifies how much time in milliseconds the traceroute should
```

File: cmd/agent/dist/conf.d/network_path.d/conf.yaml.example [hunk 2]
```diff
@@ -36,11 +36,11 @@ instances:
     #
     # max_ttl: <PORT>
 
-    ## @param timeout - integer - optional - default: 10000
-    ## Specifies how much time the full traceroute should take
-    ## in milliseconds
+    ## @param timeout - integer - optional - default: 1000
+    ## Specifies how much time in milliseconds traceroute should
```
Comment (alai97) [inline cmd/agent/dist/conf.d/network_path.d/conf.yaml.example:40]:
```suggestion
    ## Specifies how much time in milliseconds the traceroute should
```

**Review by alai97 — APPROVED** [review]
Looks good, small copy suggestion.

**Pull request merged**